optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/quad/QuadConstraintStreamTest.java

```diff
@@ -153,6 +154,21 @@ public void filterConsecutive() {
     // If (not) exists
     // ************************************************************************
 
+    @TestTemplate
+    public void ifExists_unknownClass() {
+        assumeDrools();
+        Assertions.assertThatThrownBy(() -> buildScoreDirector((factory) -> {
```

Comment:
Static imports are preferred for assertions, mock methods, etc. in tests. I know this is not enforced and what's not enforced that's allowed but I think it's a good idea to follow some extra rules even they're not enforced and more importantly be consistent about them.